Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -3169,26 +3148,12 @@ - (NSArray *)webSelectionRects
 
 - (CGPoint)_locationForGesture:(UIGestureRecognizer *)gestureRecognizer
 {
-    auto location = [gestureRecognizer locationInView:self];
-
-#if HAVE(UIKIT_WITH_MOUSE_SUPPORT)
-    if (WKHoverPlatterDomain.rootSettings.enabled)
-        return [_hoverPlatter adjustedPointForPoint:location];
-#endif
-
-    return location;
+    return [gestureRecognizer locationInView:self];
```

Comment:
Nit - would it be simpler to revert to using `[gestureRecognizer locationInView:self];`?

Reply:
A fair point! I will do that. (On the other hand, it is a handy override point. But not one we often need.)